tox.ini

```diff
@@ -1,6 +1,9 @@
 [tox]
 envlist = py27, py34, py35
 
+[testenv:docs]
+commands = python setup.py build_sphinx
```

Comment:
by breaking docs, you mean not be able to generate it right? but what if somebody changes docs but forget to check them in? we won't catch that here, right?